### DISCUSSION TITLE
Audio Quality: Am I Missing Something?

### DISCUSSION THREAD

**Discussion by TechMitten** [KRTirtho/spotube#993] [Q&A]
I seem to be missing something or encountering an issue while using the app on my Android device. The audio quality is noticeably worse than on YouTube itself, sounding very tinny, almost as if it's being played from a tin can. The reason I haven't moved on yet is that, after thoroughly checking both open and closed issues on GitHub and reviewing feedback from various sources, I haven't come across any similar reports regarding the audio quality issues I'm experiencing. The closest complaint I found was on GitHub [here](https://github.com/KRTirtho/spotube/issues/577), where the developer responded that the audio quality matches whatever is on YouTube and is not altered. However, the quality I'm hearing is significantly worse than YouTube. It almost feels like the audio is coming from the wrong speaker, but I'm not entirely sure. Just to clarify, I'm referring to streaming on Spotube, **not** downloading.

**Here are the things I've tried:**

- Changing to the Piped source (which resulted in an error).
- Adjusting the audio quality (which seems to have no effect).
- Switching the streaming media codec to M4A (with no noticeable difference).
- Changing the audio source to Jiosaavn (again, no difference).
- Clearing the app cache.
- Restarting my phone.
- Checking for updates.

I'm running Android 14 on a Pixel 6a with the latest updates for both the app and the Android OS.
App version 3.4.0 build 27.

Any help would be greatly appreciated! Thank you.

**Comment by KRTirtho** [comment]
Are you using your phone speaker or any other device?

**Comment by TechMitten** [comment]
Thanks for getting back to me. I'm using my phone speaker and I have no problem with any other audio that I play on my phone.
________________________________

**Comment by TechMitten** [comment]
I'm almost positive it's using the phones earpiece as a speaker instead of the correct speaker
